bookstore/clone.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) Jupyter Development Team.
+Distributed under the terms of the Modified BSD License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="google" content="notranslate" />
+    <meta http-equiv="Content-Language" content="en" />
+
+    <title>Clone Landing page</title>
+    <script>
+      window.onload = function() {
+        var button = document.getElementById("mybutton");
+        button.onclick = function(event) {
+          var post_json = {
+            s3_bucket: "{{ s3_bucket }}",
+            s3_key: "{{ s3_key }}"
+          };
+          var url = "{{ base_uri }}/api/bookstore/cloned/";
+          var r = document.cookie.match("\\b" + "_xsrf" + "=([^;]*)\\b");
+          var xsrf_token = r ? r[1] : undefined;
+          var xhr = new XMLHttpRequest();
+          xhr.onload = function(e) {
+            if (xhr.readyState === 4) {
+              if (xhr.status === 200) {
+                console.log(xhr.responseText);
+              } else {
+                console.error(xhr);
+              }
+            }
+          };
+          xhr.onerror = function(e) {
+            console.error(xhr.statusText);
+          };
+
+          xhr.open("POST", url, true);
+          xhr.withCredentials = true;
+          xhr.setRequestHeader("X-XSRFToken", xsrf_token);
+          xhr.send(post_json);
```

Comment:
@stormpython I'm sure I'm approaching this in a unnecessarily complicated way and would really appreciate any comments on how to simplify this (note… it doesn't work right now at all… :D)

Reply:
Ah — I got it working in the latest iteration.